client/pages/editntc/editntc.js

```diff
@@ -1,74 +1,128 @@
 // pages/userinfo/userinfo.js
+const qcloud = require('wafer2-client-sdk');
+const app = getApp();
+var util = require('../../utils/util.js');
 Page({
-
-  formSubmit: function (e) {},
-
-  bindTimeChange(e) {
-    this.setData({
-      time: e.detail.value
-    })
-  },
-
-  /**
-   * 页面的初始数据
-   */
   data: {
-
+    multiArray: [['Saturday', 'Sunday'], [10, 11, 12, 13, 14, 15, 16], ['00', 10, 20, 30, 40, 50]], // 0 was changed to "00"
+    objectMultiArray: [
```

Comment:
nit: the meaning of multiArray, multiIndex and objectMultiArray is not clear. we should have better variable names.